pygbm/gradient_boosting.py

```diff
@@ -60,9 +60,9 @@ def _validate_parameters(self):
         if self.max_iter < 1:
             raise ValueError(f'max_iter={self.max_iter} must '
                              f'not be smaller than 1.')
-        if self.n_iter_no_change < 2:
+        if self.n_iter_no_change is not None and self.n_iter_no_change < 0:
             raise ValueError(f'n_iter_no_change={self.n_iter_no_change} '
-                             f'must not be smaller than 2.')
+                             f'must be positive.')
```

Comment:
I think it now makes sense to allow `1` here, since the strategy has changed from before. Before #65 , setting it to `1`  would compare `scores[-1]` with itself so it didn't make sense, but now it would compare `scores[-2]` with `scores[-1]`.

Reply:
But we can forbid 0 maybe?

Reply:
Alright. We can keep 0 == disabled early stopping in that case.